boards/arm/nrf52_pca20020/board.c

```diff
@@ -12,6 +12,8 @@
 #define VDD_PWR_CTRL_GPIO_PIN 30
 #define CCS_VDD_PWR_CTRL_GPIO_PIN 10
 
+#ifdef CONFIG_GPIO
+
```

Comment:
This change does not seem to be related to TWI/TWIM shims or I2C DTS entries, so it should not be part of this PR.
And is it needed at all?

Reply:
On second thought, I think this issue should be solved in a different way. The GPIO module should be explicitly enabled in the default configuration for pca20020 (previously, it was enabled implicitly by the old I2C driver), so that the code in `board.c` could turn on the VDD power rail. And this power rail is needed for example for the SX1509B GPIO expander to operate, and this expander is enabled by default for this board (and it depends on GPIO as well). Thus, I think the assumption of the author of this part of code was that GPIO should be always enabled (hence, CONFIG_GPIO was not checked). @tkln Could you confirm my understanding is correct?

Reply:
Done